Ginger/GingerCoreNET/GingerCoreNET.csproj

```diff
@@ -311,6 +309,9 @@
     <Reference Include="ALM_Common">
       <HintPath>..\GingerCore\DLLs\ALM_Common.dll</HintPath>
     </Reference>
+    <Reference Include="Ginger.ExecuterService.Contracts">
+      <HintPath>..\..\..\..\Ginger.ExecuterService.Contracts\Ginger.ExecuterService.Contracts\bin\Debug\netstandard2.0\Ginger.ExecuterService.Contracts.dll</HintPath>
```

Comment:
please change it to refer it from nugget instead of direct DLL reference

Reply:
changed it to refer from the nugget and updated its version to the latest version published(4.5.3)